frontend/src/App.js

```diff
@@ -1,7 +1,38 @@
+import MainRouter from './routes'
 import React from "react";
+import Client from './client';
+import Restaurant from './restaurant';
+import Recipe from './recipe';
+import Home from './home';
 
+class App extends Component {
+    render() {
+        return (
+            <BrowserRouter>
```

Comment:
You are using imports that you have NOT imported in this file? You imported them in another file where you are not using them --> see comments below. If you look at your console, your code should not be compiling and providing you with enough information to solve the small issues that you have. You're nearly there. Be confident and read the error messages. I'm sure you can sort this out 😃